UserGuide.md

```diff
@@ -0,0 +1,15 @@
+Feature: Make a post anonymous
+How to use the feature:
+    Making a post anonymous:
+    - make a new post
```

Comment:
should probably mention that it works on existing posts too (or at least I think it does)

Reply:
changed wording in newest commit